spec/01-unit/01-db/01-schema/03-typedefs_spec.lua

```diff
@@ -175,5 +175,15 @@ describe("typedefs", function()
     assert.falsy(Test:validate({ f = 123 }))
   end)
 
+  it("headers", function()
```

Comment:
Mind making this test's name a bit more explicit? E.g. `"rejects 'host' but accepts 'host' substring"`?